src/navigation/MainStackNavigator.tsx

```diff
@@ -31,13 +33,18 @@ const MainStackNavigator = () => {
       <Stack.Screen
         name={screen_names.SPEAKERS}
         component={SpeakersScreen}
-        options={{
+        options={({navigation}: {navigation : any}) => ({
+          headerShadowVisible : false ,
           headerTitleAlign : "left" ,
           headerTitleStyle : {
             fontFamily : fonts.MONTSERRAT_REGULAR
           } ,
-          headerLeft : SpeakersBackButton ,
-        }}
+          headerLeft : () => (
+          <TouchableOpacity style={{marginRight : 8 , marginBottom : 8 }} onPress= {() => navigation.goBack()}>
```

Comment:
Good work adding `navigation.goBack()` function.
Please define the styles `marginRight : 8 , marginBottom : 8` separately from the TouchableOpacity component